### ISSUE TITLE
chainparams configs research & verification

### ISSUE THREAD

**Issue by jmgilbert2** [energicryptocurrency/energi#18]
There are many parameters in the chainparams constructor the we still need to figure out and configure. 

Among those are the various start block heights, the key pairs, the dns seeds, the checkpoint data, the base 58 prefixes. I also see 3 public keys that probably need changing:

vAlertPubKey
strSporkPubKey
strMasternodePaymentsPubKey

From a look at the code, I believe consensus.nBudgetPaymentsStartBlock is being deprecated in favor of consensus.nSuperblockStartBlock

Let's not forget about the famous BIP34 variables, whatever that is (thank you Bitcoin developers for making the code obscure)

So, basically there's still lots of fun to be had in this file

**Comment by RyanLucchese** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [chainparams configs need to be finished before launching test net](https://trello.com/c/wvPRpXhn/36-chainparams-configs-need-to-be-finished-before-launching-test-net)
